File: SmartHome-Front/SmartHomeFront/src/app/models/Camera.ts
export interface Camera {
    cameraName: string;
    cameraModel: string;
    description: string;
    photos: string[];
    deviceType: string;
    canBeUsedIndoors: boolean;
    canBeUsedOutdoors: boolean;
    hasMovementDetectionSupport: boolean;
    hasPersonDetectionSupport: boolean;
}

File: SmartHome-api/src/Domain/Camera.cs
namespace Domain;

public class Camera : Device
{
    public bool CanBeUsedIndoors { get; init; }
    public bool CanBeUsedOutdoors { get; init; }
    public bool HasMovementDetectionSupport { get; init; }
    public bool HasPersonDetectionSupport { get; init; }
}
